### PR TITLE
FIX2 - amend deposit function to include db support

### PR DESCRIPTION
**Pull request by tp859**
Not sure what branch we should be merging into but I was working off revert-25-sprint5?

Would appreciate people's thoughts on this one as there's a few ways of doing it but in the end I opted to make the Account class a subclass of Customer as its methods needed access to its parent's Customer ID.

### CODE REVIEW

**Comment by shanpatt** [comment]
This looks good to me, makes sense to make it a subclass.

Think this needs merging into sprint 5 though instead of the revert branch. As far as I can tell, the revert branch was made as a copy of main when I reverted the changes for DB and login (it was created automatically so really can't say for sure, but it doesn't have the read.me or .jar that Josh added as part of his fix).

**Comment by tp859** [comment]
Sounds good, though not sure if someone deleted the sprint5 branch unless I'm missing it?

**Comment by shanpatt** [comment]
> Sounds good, though not sure if someone deleted the sprint5 branch unless I'm missing it?

I have literally no idea where the original sprint5 branch went. It was there after reverting changes because Joshua helped me pull it down, and I could see it earlier when I was trying to figure out the differences between it and the revert branch. Weird.  

I've pushed the copy of sprint5 I had locally now as sprint_5, so hopefully you can see that & we'll use that as the new sprint5 base.

**Comment by tp859** [comment]
Weird! Anyway I've amended this to merge with the sprint_5 branch, will do the same with the other one

**Review by shanpatt — APPROVED** [review]

**Review by kp861 — APPROVED** [review]

**Review by Pookbros — COMMENTED** [review]
This PR is going to delete the README file AND re-add the logins.csv file. Can you re-add one and delete the other @tp859 and commit again?

File: server/NewBank.java [hunk 1]
```diff
@@ -16,19 +16,25 @@ private NewBank() {
     }
 
     private void addTestData() {
```
Comment (Pookbros) [inline server/NewBank.java:18]: We shouldn't need this test data now right? As they will be loaded in with the db... any point of these changes?

**Review by Pookbros — CHANGES_REQUESTED** [review]
Sorry Tom. These changes for FIX2 will have to re-done with sprint_5 as the base branch.

**Pull request closed without merging**